### PR TITLE
[tests-only] Add missing sidebar unit tests

### PR DESCRIPTION
**Pull request by JammingBen**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

### CODE REVIEW

File: packages/web-app-files/tests/unit/components/SideBar/NoSelection.spec.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import NoSelection from 'web-app-files/src/components/SideBar/NoSelection.vue'
+import { defaultPlugins, shallowMount } from 'web-test-helpers'
+
+describe('NoSelection', () => {
+  it('should render the component', () => {
+    const { wrapper } = getWrapper()
+    expect(wrapper).toMatchSnapshot()
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/components/SideBar/NoSelection.spec.ts:7]:
```suggestion
    expect(wrapper.html()).toMatchSnapshot()
```

File: packages/web-app-files/tests/unit/components/SideBar/Shares/Shared/RoleItem.spec.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import RoleItem from 'web-app-files/src/components/SideBar/Shares/Shared/RoleItem.vue'
+import { mockDeep } from 'jest-mock-extended'
+import { ShareRole } from 'web-client/src/helpers/share'
+import { defaultPlugins, mount } from 'web-test-helpers'
+
+describe('RoleItem', () => {
+  it('renders the role information', () => {
+    const { wrapper } = createWrapper()
+    expect(wrapper).toMatchSnapshot()
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/components/SideBar/Shares/Shared/RoleItem.spec.ts:9]:
```suggestion
    expect(wrapper.html()).toMatchSnapshot()
```

File: packages/web-app-files/tests/unit/components/SideBar/Shares/Links/CreateQuickLink.spec.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import CreateQuickLink from 'web-app-files/src/components/SideBar/Shares/Links/CreateQuickLink.vue'
+import { defaultPlugins, mount } from 'web-test-helpers'
+
+const selectors = { createBtn: '.link-name-container button' }
+
+describe('CreateQuickLink', () => {
+  it('renders the quick link button', () => {
+    const { wrapper } = createWrapper()
+    expect(wrapper.find(selectors.createBtn).exists()).toBeTruthy()
+    expect(wrapper).toMatchSnapshot()
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/components/SideBar/Shares/Links/CreateQuickLink.spec.ts:10]:
```suggestion
    expect(wrapper.html()).toMatchSnapshot()
```

File: packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/EditDropdown.spec.ts [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+describe('EditDropdown', () => {
+  beforeEach(() => {
+    Settings.defaultZone = 'utc'
+  })
+  describe('default actions', () => {
+    it('render "remove share" and "access details" as default', () => {
+      const { wrapper } = createWrapper()
+      expect(wrapper.findAll(selectors.actionMenuItem).length).toBe(2)
+      expect(wrapper).toMatchSnapshot()
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/EditDropdown.spec.ts:25]:
```suggestion
      expect(wrapper.html()).toMatchSnapshot()
```

**Review by kulmann — CHANGES_REQUESTED** [review]

**Comment by kulmann** [comment]
@JammingBen could you revisit your PR and bring it up to date?

**Review by kulmann — APPROVED** [review]

**Review by JanAckermann — APPROVED** [review]

**Pull request merged**